### ISSUE TITLE
How to reduce the number of warnings in Apache logs

### ISSUE THREAD

**Issue by ruslan-afanasiev** [chansen/p5-http-tiny#147]
Hi,
Any idea on how to suppress such a warning in Apache error logs?

```
Use of uninitialized value in scalar assignment at /opt/perl-5.20.2/lib/site_perl/5.20.2/HTTP/Tiny.pm line 158
```
It seems it needs to set either `http_proxy` or `HTTP_PROXY` env. variable. But, I don't need it because there is no proxy in my case.
Thanks!

**Comment by xdg** [comment]
What version of HTTP::Tiny?

**Comment by ruslan-afanasiev** [comment]
Hi David 👋 
Here you are 

```
root@ip-ZZ-YY-XX:~/# PERL5LIB=/opt /opt/perl/bin/perl -MHTTP::Tiny -e 'print "$HTTP::Tiny::VERSION\n"'
0.078
```

**Comment by xdg** [comment]
You must be on a very old version of Perl.  Fixed in 3caed7f.  Released as dev release to CPAN as HTTP-Tiny-0.079-TRIAL.tar.gz.  If CPAN smokers don't find any problems, I'll do a stable release in the next day or two.

**Comment by xdg** [comment]
HTTP-Tiny-0.080.tar.gz released to CPAN.
